app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -134,6 +134,10 @@ def remove_subscriptions
       param :quantity, :number, :desc => N_("Quantity of this subscriptions to add"), :required => true
     end
     def add_subscriptions
+      if @host.organization.simple_content_access?
+        fail ::Katello::HttpErrors::BadRequest, _("This host's organization has Simple Content Access enabled.  Attaching subscriptions is neither necessary nor allowed.")
```

Comment:
Thoughts on this wording?  I'm not sure I'm happy with it.

Reply:
For auto-attach we say `This host's organization is in Simple Content Access mode. Auto-attach is disabled` so I think we can be consistent and say `This host's organization is in Simple Content Access mode. Attaching subscriptions is disabled` Concise and explanatory enough, maybe?

Reply:
@jturel looks good to me; updated.